### PR TITLE
2019/06 - Velocity San Jose

### PR DESCRIPTION
**Pull request by campoy**
Updates: #112

### CODE REVIEW

File: 2019-velocity-san-jose/analyzing-k8s.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Abstract:
+
+> In this talk, I will assess the status of the Kubernetes project and its community by analyzing its source code.
+>
+> We will present the kind of technical difficulties one needs to overcome in order to successfully analyze code at this scale (well over 2 million lines of code total), including language classification, parsing, etc.
+> 
+> Then we will cover the kind of metrics that can give us clear insights on how the project is doing, where the innovation efforts are being directed to, as well as which pieces of software seem to be the best candidates for refactoring.
+> 
+> The metrics will be mostly related to the two most common migrations we've observed in the industry: Cloud Native and Innesource efforts. Therefore, we will analyze some technical guidelines and how the source code adheres to them, but also social aspects of how different teams collaborate and whether any silos exist.
```
Comment (vcoisne) [inline 2019-velocity-san-jose/analyzing-k8s.md]: I would replace migrations by initiatives ?

File: 2019-velocity-san-jose/analyzing-k8s.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+> 
+> Although most of these metrics will be extracted thanks to source{d} Engine (sourced.tech/engine), which provides a SQL interface to an ensemble of git repositories,the takeaways are applicable to any other techniques including bash scripts powered by the git binary and lots of patience.
+
+**Notes**:
+
+> Most of the analysis has already been done and will be published as a PR piece during Kubecon.
```
Comment (vcoisne) [inline 2019-velocity-san-jose/analyzing-k8s.md]: I would not mention the PR aspect. I would just announced in a blog post to be published during KubeCon.

**Review by vcoisne — COMMENTED** [review]
LGTM with 2 minor comments.

File: 2019-velocity-san-jose/analyzing-k8s.md (hunk 1 above)
Comment (campoy) [inline 2019-velocity-san-jose/analyzing-k8s.md]:
```suggestion
> The metrics will be mostly related to the two most common initiatives we've observed in the industry: Cloud Native and Innesource efforts. Therefore, we will analyze some technical guidelines and how the source code adheres to them, but also social aspects of how different teams collaborate and whether any silos exist.
```

Comment (campoy) [reply 2019-velocity-san-jose/analyzing-k8s.md]: Ok, done with a suggestion below

File: 2019-velocity-san-jose/analyzing-k8s.md (hunk 2 above)
Comment (campoy) [inline 2019-velocity-san-jose/analyzing-k8s.md]:
```suggestion
> Most of the analysis has already been done and will be published as a blog post during Kubecon.
```

**Comment by campoy** [comment]
Ping @vcoisne

**Review by vcoisne — APPROVED** [review]
LGTM

**Pull request merged**